Doc/faq/programming.rst

```diff
@@ -1613,9 +1613,15 @@ method too, and it must do so carefully.  The basic implementation of
            self.__dict__[name] = value
        ...
 
-Most :meth:`!__setattr__` implementations must modify
-:attr:`self.__dict__ <object.__dict__>` to store
-local state for self without causing an infinite recursion.
+Many :meth:`!__setattr__` implementations call :meth:`object.__setattr__` to set
```

Comment:
I'd add a link on the first one (so that you can get taken to information on `__setattr__` methods in general) but not on the second one (since the link doesn't actually take you to docs on `object.__setattr__`, but to information on `__setattr__` methods in general)

```suggestion
Many :meth:`~object.__setattr__` implementations call :meth:`!object.__setattr__` to set
```